src/routes/game/components/dialogs/components/FiveDialog.vue

```diff
@@ -47,14 +47,20 @@ const { player } = storeToRefs(gameStore);
 const selectedCard = ref();
 const emit = defineEmits(['resolve-five']);
 
-const disableButton = computed(() => !player.value.hand.length || !selectCard.value);
+const disableButton = computed(() => player.value.hand.length && !selectedCard.value);
```

Comment:
This throws a vue warning, disabled looks for a boolean but this is returning `0`

Reply:
We can fix this by casting the length to boolean with !!
```suggestion
const disableButton = computed(() => !!player.value.hand.length && !selectedCard.value);
```